scripts/exgdas_enkf_sfc.sh

```diff
@@ -173,8 +173,8 @@ if [ $DOSFCANL_ENKF = "YES" ]; then
 
         [[ $TILE_NUM -eq 1 ]] && mkdir -p $COMOUT_ENS/$memchar/RESTART
```

Comment:
Missed "atmos", it should be 
[[ $TILE_NUM -eq 1 ]] && mkdir -p $COMOUT_ENS/$memchar/**atmos**/RESTART

Reply:
Thank you for the review and providing a fix for the bug.